Description: fix pod test errors
 "L<> starts or ends with whitespace"
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=88030
Bug-Debian: https://bugs.debian.org/720429
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=88030
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-08-22

--- a/lib/Mail/SPF/Util.pm
+++ b/lib/Mail/SPF/Util.pm
@@ -155,9 +155,9 @@
 throws I<Mail::SPF::EInvalidOptionValue>
 
 Returns the given I<NetAddr::IP> IPv4 or IPv6 address compactly formatted as a
-I<string>.  For IPv4 addresses, this is equivalent to calling L< NetAddr::IP's
-C<addr> |NetAddr::IP/addr> method.  For IPv6 addresses, this is equivalent to
-calling L< NetAddr::IP's C<short> |NedAddr::IP/short> method.  Throws a
+I<string>.  For IPv4 addresses, this is equivalent to calling L<NetAddr::IP's
+C<addr>|NetAddr::IP/addr> method.  For IPv6 addresses, this is equivalent to
+calling L<NetAddr::IP's C<short>|NetAddr::IP/short> method.  Throws a
 I<Mail::SPF::EInvalidOptionValue> exception if the specified object is not a
 I<NetAddr::IP> IPv4 or IPv6 address object.
 
